data/fields/transformer.json

```diff
@@ -4,14 +4,13 @@
     "label": "Type",
     "strings": {
         "options": {
-            "distribution": "Distribution",
-            "generator": "Generator",
-            "converter": "Converter",
-            "traction": "Traction",
-            "auto": "Autotransformer",
-            "phase_angle_regulator": "Phase Angle Regulator",
-            "auxiliary": "Auxiliary",
-            "yes": "Unknown"
+            "main": "Forwards power",
+            "distribution": "Feeds final consumers, installed outside substations",
+            "generator": "Steps-up voltage in power plants",
+            "converter": "Feeds power converters",
+            "phase_angle_regulator": "Regulates phase angle",
+            "auxiliary": "Feeds internal systems in substations",
```

Comment:
Were the previous strings incorrect? The new strings can end up being so lengthy in translation that they ideally would be displayed as tooltips, though I’m unsure if schema-builder supports localizable combo option tooltips yet. For now, perhaps each option could start with the short name, like “Distribution: Feeds final consumers, installed outside substations”.

Reply:
They were probably not incorrect, but I thought that having an almost 1:1 copy of the raw tag value isn't a big help for mappers (otherwise this field could have been a regular combo field), that's why I asked for more descriptive labels.

:+1: for adding the tag values as part of the labels, I've tweaked it in a33d777 – does that look better?